UseSecurityFeatures.md

```diff
@@ -117,7 +138,8 @@ Run the following command to build and flash the demo project:
 ```
 idf.py -p PORT flash monitor
```

Comment:
Shouldn't this be encrypted-flash here as well?
Isn't the point of the above to use encrypted-flash for this build?

Reply:
no, the first one does not need the encrypt in it.
The subsequent ones need it.  It is stated in the comment.

Reply:
Got it, thanks!